app/decorators/physical_server_decorator.rb

```diff
@@ -0,0 +1,14 @@
+class PhysicalServerDecorator < MiqDecorator
+  def img_for_physical_vendor
+    "svg/vendor-#{label_for_vendor.downcase}.svg"
+  end
+
+  def img_for_health_state
```

Comment:
**But**: if you're creating a new decorator, please make sure it has one or both of the `fonticon` and `fileicon` methods, which return the right thing for that entity.